__tests__/bin/vip-config-envvar-delete.js

```diff
@@ -1,16 +1,19 @@
+// @flow
+
 /**
  * External dependencies
  */
+import { describe, expect, it, jest } from '@jest/globals';
```

Comment:
Flow is unaware of Jest; wherever we have to add the `@flow` annotation, we also have to import this file.